scripts/linters/css_linter_test.py

```diff
@@ -0,0 +1,172 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/linters/css_linter.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import subprocess
+
+from core.tests import test_utils
+import python_utils
+
+from . import css_linter
+
+PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+CONFIG_PATH = os.path.join(
+    PARENT_DIR, 'oppia', 'core', 'templates', 'css', '.stylelintrc')
+
+LINTER_TESTS_DIR = os.path.join(os.getcwd(), 'core', 'tests', 'linter_tests')
+VALID_CSS_FILEPATH = os.path.join(LINTER_TESTS_DIR, 'valid.css')
+INVALID_CSS_FILEPATH = os.path.join(LINTER_TESTS_DIR, 'invalid.css')
+
+
+def appears_in_linter_stdout(phrases, linter_stdout):
```

Comment:
This looks like a duplicate function, can we use assertSetEqual or assertSequenceEqual? If no, then change the name of this function to assert_same_list_elements and move it to test_utils.

Reply:
Done.

Reply:
Which you have used `assertSetEqual or assertSequenceEqual` or a separate function? If the later one then why, can you explain?

Reply:
Because we are not checking just checking a sequence, we are checking parts of a message.
For example: `home/opensource/oppia/scripts\n 18:16 x This line has an error    error-name`
we are only checking parts, we are passing a list and checking that list `['18:16', 'This line has an error']`